c2corg_ui/views/xreport.py

```diff
@@ -20,6 +22,29 @@ def redirect_to_full_url(self):
 
     @view_config(route_name='xreports_view')
     def detail(self):
```

Comment:
The feed shown in the profile pages are handled using the feed directive. There is no feed caching managed by the profile UI services. The UI server side caching is not even aware feed items are loaded through the profile data template.

I don't think there should be a difference in the code of profile's ``profile`` and xreport's ``detail`` views.